model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/PushMessageInformation.java

```diff
@@ -36,6 +36,10 @@
     private Date submitDate = new Date();
     private long totalReceivers;
 
+    private long appOpenCounter;
+    private Date firstOpenDate;
```

Comment:
that is indeed a bug

the idea is to store the timestamp. and indeed a query gets us what we want
(oldest and youngest timestamp)

On Tue, Apr 14, 2015 at 1:55 PM, Erik Jan de Wit <notifications@github.com>
wrote:

> In
> model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/PushMessageInformation.java
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/523#discussion_r28319943>
> :
>
> > @@ -36,6 +36,10 @@
> >      private Date submitDate = new Date();
> >      private long totalReceivers;
> >
> > +    private long appOpenCounter;
> > +    private Date firstOpenDate;
>
> Wouldn't it be better to get this information from a query instead of
> duplicating it, it will not be true once updated
> [image: metrics]
> <https://cloud.githubusercontent.com/assets/51133/7136140/dfe1ad60-e2ad-11e4-823a-b44815a54256.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/523/files#r28319943>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf